src/main/java/com/amazon/ion/impl/macro/EExpressionArgsReader.java

```diff
@@ -237,7 +237,7 @@ protected void readValueAsExpression(boolean isImplicitRest, List<Expression.EEx
         }
         IonType type = reader.encodingType();
         List<SymbolToken> annotations = getAnnotations();
-        if (isImplicitRest) {
+        if (isImplicitRest && !isContainerAnExpressionGroup()) {
```

Comment:
This is a bug fix. Without this, an expression group as the final parameter to a rest-compatible argument would be parsed into a nested expression group, which is illegal.

Reply:
This is one of the bugs that I noticed yesterday. Thanks for catching and fixing this.